lib/chromedriver.js

```diff
@@ -62,7 +62,7 @@ class Chromedriver extends events.EventEmitter {
     if (this.logPath) {
       args = args.concat([`--log-path=${this.logPath}`]);
     }
-
+    
```

Comment:
Yes, It did pass lint part of test locally. But It didn't pass unit-test. And I'm still finding the problem
![image](https://user-images.githubusercontent.com/4971494/29618901-1f0efd8a-884c-11e7-8ada-cd269a55e5e6.png)